index.html

```diff
@@ -385,23 +384,21 @@ <h4>Primary Language</h4>
 
           <section>
             <h4>Script Events</h4>
-            <p>The <dfn>Script Events</dfn> part of the model provides the list of intervals corresponding to dialogue, on screen text, or descriptions.</p>
+            <p>A <a>DAPT Script</a> MAY contain zero or more <a>Script Event</a> objects, each corresponding to dialogue, on screen text, or descriptions for a given time interval.</p>
           </section>
 
           <section>
             <h4>Characters</h4>
-            <p>The <dfn>Characters</dfn> part of the model provides the list of characters referenced in the <a>Script Events</a>.</p>
+            <p>A <a>DAPT Script</a> MAY contain zero or more <a>Character</a> objects, each describing a character referenced in a <a>Script Event</a>.</p>
           </section>
-
-          <p class="issue" data-number="14"></p>
       </section>
       <section>
         <h3>Character</h3>
 
         <p>In <a>Dubbing Scripts</a>, it is necessary to identify each character in the programme. This is done with a <dfn>Character</dfn> object which has the following properties: 
           <ul>
-            <li><dfn data-lt="Character Identifier">Identifier</dfn> which is a unique identifier to indicate when a <a>Character</a> participates in a <a>Script Event</a></li>
-            <li><dfn data-lt="Character Name">Name</dfn> which is the name of the <a>Character</a> in the programme</li>
+            <li>a mandatory <dfn data-lt="Character Identifier">Identifier</dfn> which is a unique identifier to indicate when a <a>Character</a> participates in a <a>Script Event</a></li>
```

Comment:
Noticing that the phrasing here should be adjusted (could be a separate issue but may as well deal with it here for expediency): the identifier does not indicate when a Character participates in a Script Event; rather, the identifier can be used to reference a Character.

```suggestion
            <li>a mandatory <dfn data-lt="Character Identifier">Identifier</dfn>
            which is a unique identifier used to reference the character from elsewhere in the document,
            for example to indicate when a <a>Character</a> participates in a <a>Script Event</a>,
            or to link a <a>Character Style</a> to its <a>Character</a>.</li>
```

Reply:
But the Character should be referenced only when it participates in the event, no?

Reply:
We need to make sure that the requirement is expressed in the TTML section.